### PR TITLE
add nullability to prismaNode

### PR DESCRIPTION
**Pull request by MaxAst**

### CODE REVIEW

File: packages/plugin-prisma/src/schema-builder.ts [hunk 1]
```diff
@@ -105,14 +106,14 @@ schemaBuilderProto.prismaNode = function prismaNode(
...
-      const record = await (delegate.findUniqueOrThrow
+      const record = await (delegate.findUniqueOrThrow && !nullable
         ? delegate.findUniqueOrThrow({
             ...query,
             where: rawFindUnique ? rawFindUnique(id, context) : { [fieldName]: idParser!(id) },
           } as never)
         : delegate.findUnique({
             ...query,
-            rejectOnNotFound: true,
+            rejectOnNotFound: nullable ? false : true,
```
Comment (hayes) [inline packages/plugin-prisma/src/schema-builder.ts:116]: `...(nullable ? {} : { rejectOnNotFound: true },`

**Comment by hayes** [comment]
Looks good, would be great to get something added to the docs and add a changeset. 

Thanks for adding this feature!

File: packages/plugin-prisma/tests/nodes.test.ts [hunk 2]
```diff
@@ -90,7 +90,66 @@ describe('prisma', () => {
     `);
   });
 
-  it('query for single node', async () => {
+  it.only('query for nullable single node', async () => {
```
Comment (hayes) [inline packages/plugin-prisma/tests/nodes.test.ts:93]: need to revert this `it.only`

**Comment by hayes** [comment]
@MaxAst looks like some tests are failing, I haven't had time to dig into it yet, but I think it's other tests using the prismaNode you modified (there are lots of tests that assert the specific prisma calls being made, and it looks like the failing tests are seeing findUnique instead of findUniqueOrThrow)

**Comment by MaxAst** [comment]
> @MaxAst looks like some tests are failing, I haven't had time to dig into it yet, but I think it's other tests using the prismaNode you modified (there are lots of tests that assert the specific prisma calls being made, and it looks like the failing tests are seeing findUnique instead of findUniqueOrThrow), which likely just need to be updated

@hayes my bad! just saw you fixed them already, thank you!

**Pull request merged**